core/block_validator.go

```diff
@@ -114,26 +93,6 @@ func (v *BlockValidator) ValidateBody(block *types.Block) error {
 	if err := v.ValidateL1Messages(block); err != nil {
 		return err
 	}
-	if v.checkCircuitCapacity {
-		// if a block's RowConsumption has been stored, which means it has been processed before,
-		// (e.g., in miner/worker.go or in insertChain),
-		// we simply skip its calculation and validation
-		if rawdb.ReadBlockRowConsumption(v.bc.db, block.Hash()) != nil {
-			return nil
-		}
-		rowConsumption, err := v.validateCircuitRowConsumption(block)
-		if err != nil {
-			return err
-		}
-		log.Trace(
-			"Validator write block row consumption",
-			"id", v.circuitCapacityChecker.ID,
-			"number", block.NumberU64(),
-			"hash", block.Hash().String(),
-			"rowConsumption", rowConsumption,
-		)
-		rawdb.WriteBlockRowConsumption(v.bc.db, block.Hash(), rowConsumption)
```

Comment:
this is missing rn

Reply:
https://github.com/scroll-tech/go-ethereum/pull/894/commits/c666dacc68d4b10913b94d2be97dcf3496065eee